### ISSUE TITLE
nohup information

### ISSUE THREAD

**Issue by slwgsc** [ultralytics/ultralytics#21164]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I use nohup command for training, but get too many training information which is not the same as console output. How to simplify the log file with one line training information each epoch?

![Image](https://github.com/user-attachments/assets/63b0cbb0-0351-4bd3-85f1-4d3d779dde98)
![Image](https://github.com/user-attachments/assets/709c7dc4-9595-4ec9-8859-824e29738103)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @slwgsc, thank you for reaching out and using Ultralytics 🚀! This is an automated response to help guide you while an Ultralytics engineer reviews your question and will assist you soon.

For new users, we suggest checking the [Docs](https://docs.ultralytics.com/) where you’ll find helpful [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. Many common questions are already answered there.

If your issue is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us investigate further.

For custom training or logging ❓ questions like this, please share as much detail as possible—such as specific command lines, logging configs, or relevant snippets—to help us better understand your request.

Join the Ultralytics community for support and discussion! For real-time chat, join [Discord](https://discord.com/invite/ultralytics) 🎧. For in-depth topics, visit [Discourse](https://community.ultralytics.com/). To connect with fellow users, check out our [Subreddit](https://www.reddit.com/r/Ultralytics/).

## Upgrade

Make sure you’re using the latest `ultralytics` package with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to ensure your issue isn’t already resolved:

```bash
pip install -U ultralytics
```

## Environments

YOLO runs smoothly in these verified environments (with all dependencies, including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you for your patience while our team reviews your question!

**Comment by glenn-jocher** [comment]
You can reduce the log verbosity by setting `verbose=False` in your training command, which will suppress the detailed progress bars and show only essential information per epoch:

```python
from ultralytics import YOLO
model = YOLO('yolo11n.pt')
model.train(data='your_data.yaml', epochs=100, verbose=False)
```

Alternatively, you can redirect stderr to suppress progress bars while keeping stdout: `nohup python train.py 2>/dev/null > output.log &`
